### DISCUSSION TITLE
HasManyRepeater: How do I change the position up/down?

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#1023] [Help] [answered]
Hello, Is it possible to change the position of rows inside a HasManyRepeater? I didn't find anything in the documentation regarding this. Thanks.

**Comment by danharrin** [comment, marked as answer]
Hey, we disabled the item movement for the `HasManyRepeater` as the order is not stored the database like with the normal JSON `Repeater`.

**Reply by ghost** [reply]
Wouldn't it be very useful to have the order for `HasManyRepeater` too ? Since the child has a position column.

**Reply by danharrin** [reply]
It shouldn't be default behaviour as most relationships don't have an ordering column, but feel free to PR this feature and I will merge it.

**Reply by ghost** [reply]
@danharrin Since I'm not expert PHP dev, I would like to sponsor this feature. Is it possible to get in touch to send me a quote regarding it? Thanks.

**Reply by danharrin** [reply]
If you'd like to sponsor me through GitHub (however much you can comfortably afford), I'll take a look at the feature later :)

**Reply by romaldyminaya** [reply]
I have this need in a project, I'll be working in a solution in few hours to implement this.

@danharrin basically the HasManyRepater in order to work, just has to make sure that the relation has an order column and save properly right?

**Reply by danharrin** [reply]
I think you should be able to define your own order column name, and if it's defined then the item movement is enabled and the order is saved.

**Reply by danharrin** [reply]
#1978
